File: 1781-check-if-two-string-arrays-are-equivalent/check-if-two-string-arrays-are-equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        s= ""
        for i in word1:
            s = s+i
        k=""
        for j in word2:
            k = k+j
        if(s == k):
            return True
        else:
            return False

File: 1781-check-if-two-string-arrays-are-equivalent/check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string s = "";
        string k = "";
        for(auto i:word1){
            s+=i;
        }
        for(auto j:word2){
            k+=j;
        }
        return s == k;
    }
};